site/content/integrate/apps/_index.md

```diff
@@ -1,41 +1,85 @@
 weight: 20
 ---
 
-Apps are lightweight, interactive add-ons to Mattermost which can be hosted as HTTP services, or as serverless functions on AWS Lambda, OpenFaaS, Kubernetes, etc. to run without dedicated infrastructure. Apps can:
+If you're a developer or system integrator looking to learn how to extend Mattermost product functionality, but not sure where to start,  you're in the right place. In this section, you'll learn how to get started developing Mattermost Apps using the Mattermost Apps Framework.
 
-- Display interactive, dynamic modal forms and message actions.
-- Be written in any language.
-- Attach themselves to locations in the Mattermost user interface (e.g. channel bar buttons, post menu, channel menu, commands), and can add their custom slash commands with full autocomplete.
-- Receive webhooks from Mattermost, and third-parties, and use the Mattermost REST APIs to post messages, etc.
-- Work on both Mobile and Desktop clients so developers can focus on the functionality of their apps.
-- Be deployed using our serverless hosting infrastructure keeping data secure and supporting scalability by being stateless.
-- Include dynamic fields, code as an on-demand function, and interactive modals.
+- Is JavaScript or Golang your language of choice? Jump right in to our [JavaScript](https://developers.mattermost.com/integrate/apps/quick-start-js/) and [Golang](https://developers.mattermost.com/integrate/apps/quick-start-go/) developer quick start guides. Looking for another language? *[learn how to contribute here]*
+- Got an existing Mattermost plugin you want to convert to an app? *[where to go for that info?]*
 
-When you develop using the Apps Framework, your apps can:
+### Want to extend Mattermost **without** having to develop an app? 
+Visit the [Mattermost Development Guides](https://developers.mattermost.com/integrate/admin-guide/) to explore the many Mattermost integrations already available. Then, browse the [Mattermost Marketplace](https://mattermost.com/marketplace/) to learn how our Mattermost Community is helping extend core Mattermost features with the Apps Framework to fit their specific organizational needs.
 
-- Create slash commands
+## What are Mattermost Apps?
+Mattermost Apps are lightweight, interactive add-ons to Mattermost implemented as a collection of HTTP endpoints. You can write Mattermost apps in your language of choice, and your app is supported on all Mattermost clients, including web, desktop, and mobile.
 
-![image](app-slash-command-zoomed-in.png)
+Apps are available as a Developer Preview and we're looking for your feedback! Share constructive feedback in the [Mattermost Apps channel](https://community.mattermost.com/core/channels/mattermost-apps) on our Mattermost Community instance.
 
-- Post messages to channels
+## What's the Mattermost Apps Framework?
 
-![image](app-bot.png)
+*[what is it and how does it differ from the existing plugin framework?]*
```

Comment:
App framework wraps the Mattermost API into a tool making it easy to develop integrations that interact with the Mattermost Server and Web App in predefined ways. 

They are different than plugins in that plugins have the ability to modify the Mattermost Server and Web App without having to fork those specific repositories.